app/(candidate)/dashboard/website/util/website.util.js

```diff
@@ -10,7 +11,11 @@ export const WEBSITE_STATUS = {
   unpublished: 'unpublished',
 }
 
-export function getWebsiteUrl(vanityPath, preview = false) {
+export function getWebsiteUrl(vanityPath, preview = false, domain = {}) {
+  if (domain?.name && isDomainActive(domain)) {
+    return `https://${domain.name}`
```

Comment:
Do we _need_ the protocol hard-coded here?

`<Link />` should be able to handle rendering a link w/o it, and Vercel and AWS's web service layer should be able to handle the unsecure-secure redirect seamlessly for the user...